src/CashuWallet.ts

```diff
@@ -331,12 +332,14 @@ class CashuWallet {
 	/**
 	 * Requests a mint quote form the mint. Response returns a Lightning payment request for the requested given amount and unit.
 	 * @param amount Amount requesting for mint.
+	 * @param description optional description for the mint quote
 	 * @returns the mint will return a mint quote with a Lightning invoice for minting tokens of the specified amount and unit
 	 */
-	async createMintQuote(amount: number) {
+	async createMintQuote(amount: number, description?: string) {
 		const mintQuotePayload: MintQuotePayload = {
 			unit: this._unit,
-			amount: amount
+			amount: amount,
+			description: description || null
```

Comment:
Is the description field mandatory on future mint requests? If not, I would prefer to not pass the null value here. If left undefined, description will be dropped from the payload completely, while if set null it will be `{"description":null}`